src/model/Layer.ts

```diff
@@ -15,6 +15,7 @@ export interface DefaultLayerSourceConfig {
   resolutions?: number[];
   attribution?: string;
   requestParams?: DefaultRequestParams;
+  bearerToken?: boolean;
```

Comment:
The name `bearerToken` implies an actual token value. I would suggest to rename it to `useBearerToken` (or similiar).